docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -115,6 +115,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 ### Repository - Release (`repository.release`)
 
 - `ALLOWED_TYPES`: **\<empty\>**: Comma-separated list of allowed file extensions (`.zip`), mime types (`text/plain`) or wildcard type (`image/*`, `audio/*`, `video/*`). Empty value or `*/*` allows all types.
+- `DEFAULT_PAGING_NUM`: **10**: The default paging number of releases user interface
```

Comment:
```suggestion
- `DEFAULT_PAGING_NUM`: **10**: The default paging number of releases in the user interface
```

Reply:
Is there a reason why this does not apply to the API as well?
Or is the API not paginated?

Reply:
api has it's own dedicated limit settings for all paginated endpoints